modules/options/src/main/scala/scala/build/Artifacts.scala

```diff
@@ -395,7 +371,7 @@ object Artifacts {
         (d, p, a, os.Path(f, Os.pwd))
       },
       extraClassPath,
-      extraCompileOnlyJars ++ extraStubsJars,
+      extraCompileOnlyJars,
```

Comment:
Is `stub` module is not used, we can remove this module from build.sc - [stub-module](https://github.com/VirtusLab/scala-cli/blob/112c02aa0f2029bc0fb7e010cd54da4536e076f1/build.sc#L62)